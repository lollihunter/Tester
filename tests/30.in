4 20 20
1 1
1 20
3 7
2 17
4 19
1 13
1 11
2 6
3 1
4 5
3 20
1 19
1 12
2 4
4 17
1 18
3 10
3 3
2 18
4 20
