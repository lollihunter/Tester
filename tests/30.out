10
1 18 10 3 17 13 6 4 19 20 
